cdap-ui/app/cdap/components/Home/index.js

```diff
@@ -142,7 +143,7 @@ export default class Home extends Component {
         if (!isValid) {
           this.eventEmitter.emit(globalEvents.PAGE_LEVEL_ERROR, {
             statusCode: 404,
-            data: `Namespace '${namespace}' does not exist.`,
+            data: GLOBALS.pageLevelErrors['INVALID-NAMESPACE'](namespace),
```

Comment:
Should we define 'INVALID-NAMESPACE' as a constant since it's used across multiple places?

Reply:
Since it is only referenced once in each place, I am not sure it is necessary since it won't necessarily simplify the code. How do you think it would be helpful?